03-make/make_simplest/Makefile

```diff
@@ -1,9 +1,11 @@
-
+# blank line is better to be removed
 hello: main.o
 	gcc -o hello main.o -lc
-
-main.o: main.c
+main.o:
+#main.c -removed from 'main.o: main.c' line
```

Comment:
What is the reason of removing main.c from dependencies?

Reply:
Because there is no function with such name in the Makefile. This is not an issue to leave it in case we're going to add this function later. But currently I prefer to keep the code clean.

Reply:
@yekovalyov is this good?

Reply:
Make tries to analyze all dependencies and updates the project by compare date of target and dependencies. If You remove main.c from dependency list make will not rebuild main.o after changes in main.c

Reply:
> Because there is no function with such name in the Makefile. 

main.c is file name. It is the source file for target file main.o

Reply:
@yekovalyov I got it. 
Please approve to merge this?